project/src/components/card-film/card-film.tsx

```diff
@@ -9,18 +11,22 @@ type CardFilmProps= {
 }
 
 function CardFilm({ card, setTarget, filmCardTarget }: CardFilmProps) {
+  const [isFosued, setIsFocused] = useState(false);
+
   const handleMouseEnter = () => {
     setTarget(card.id);
+    setIsFocused(!isFosued);
   };
 
   const handleMouseLeave = () => {
     setTarget(undefined);
+    setIsFocused(!isFosued);
   };
 
   return (
-    <article className="small-film-card catalog__films-card" onMouseEnter={handleMouseEnter} onMouseLeave={handleMouseLeave}>
+    <article className="small-film-card catalog__films-card" id={`${card.id}`} onMouseEnter={handleMouseEnter} onMouseLeave={handleMouseLeave}>
       <div className="small-film-card__image">
-        <img src={card.img} alt={card.title} width="280" height="175" />
+        {isFosued ? <VideoPlayer img={card.img} video={card.videoLink}></VideoPlayer> : <img src={card.img} alt={card.title} width="280" height="175" />}
```

Comment:
Строка нечитаема становится лучше разносить на несколько строк
```suggestion
        {isFosued
           ? <VideoPlayer img={card.img} video={card.videoLink}></VideoPlayer>
           : <img src={card.img} alt={card.title} width="280" height="175" />
         }
```